spec/rails/cacheable_method_system_spec.rb

```diff
@@ -0,0 +1,10 @@
+describe "Fortitude cacheable method behavior in Rails", :type => :rails do
+  uses_rails_with_template :cacheable_method_system_spec
+
+  it "caches properly" do
+    expect_match("localization?locale=en", /hello is: hello 1/)
+    expect_match("localization?locale=en", /hello is: hello 1/)
+    expect_match("localization?locale=fr", /hello is: bonjour 2/)
+    expect_match("localization?locale=fr", /hello is: bonjour 2/)
+  end
```

Comment:
This test kinda sucks. I'd like to test some of this stuff with a system spec (i.e. a non-integration test), but those don't appear to require Rails.